### PR TITLE
[POAE7-2526] Fix Non Groupby Agg Arrow Format null value.

### PR DESCRIPTION
**Pull request by haiweizh**
### What changes were proposed in this pull request?

Fix Non Groupby Agg Arrow Format null value.

### Why are the changes needed?

Fix Non Groupby Agg Arrow Format null value.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Uts.

### Which label does this PR belong to?

BUG,cider,test

### CODE REVIEW

**Comment by haiweizh** [comment]
Plz help to take a review. @jikunshang @yma11 @kd97

File: cider/exec/module/batch/CiderBatchUtils.cpp [hunk 1]
```diff
@@ -43,9 +43,9 @@ namespace CiderBatchUtils {
     C_TYPE* null_buf = (C_TYPE*)table_ptr[i * 2];                           \
     for (auto j = 0; j < table_row_num; ++j) {                              \
       C_TYPE value = buf[j];                                                \
-      bool is_null = null_buf[j];                                           \
+      bool is_not_null = null_buf[j];                                       \
```
Comment (winningsix) [inline cider/exec/module/batch/CiderBatchUtils.cpp:46]: Hmm, why do we need to create ```is_not_null``` variable? Just set value to ```null_data``` directly.

Comment (haiweizh) [reply cider/exec/module/batch/CiderBatchUtils.cpp:46]: should convert null sign value from C_TYPE to bool.

File: cider/exec/module/batch/CiderBatchUtils.cpp [hunk 2]
```diff
@@ -57,14 +57,13 @@ namespace CiderBatchUtils {
     std::vector<bool> null_data;                                              \
     col_data.reserve(table_row_num);                                          \
     null_data.reserve(table_row_num);                                         \
-    null_data.push_back(false);                                               \
     C_TYPE* buf = (C_TYPE*)table_ptr[i];                                      \
     C_TYPE* null_buf = (C_TYPE*)table_ptr[i * 2];                             \
     for (auto j = 0; j < table_row_num; ++j) {                                \
       C_TYPE value = buf[j];                                                  \
-      bool is_null = null_buf[j];                                             \
+      bool is_not_null = null_buf[j];                                         \
```
Comment (harborn) [inline cider/exec/module/batch/CiderBatchUtils.cpp:64]: in this macro, C_TYPE can't be `bool`
so this line should be: `bool is_not_null = (null_buf[j] != 0);` ?

Comment (haiweizh) [reply cider/exec/module/batch/CiderBatchUtils.cpp]: Done.

**Review by yma11 — APPROVED** [review]

**Review by winningsix — APPROVED** [review]

**Pull request merged**